rotkehlchen/api/rest.py

```diff
@@ -3910,9 +3911,9 @@ def _query_kraken_staking_events(
             Location.KRAKEN,
         )
         message = ''
-        if exchanges_list is None:
-            events = []
-        else:
+        events: List[StakingEvent] = []
+        total_entries = 0
```

Comment:
Why initialize this to 0? There is no need. It gets assigned to later on

```suggestion
```

Reply:
Is initialized to 0 because if there is no kraken instances then this variable never gets assigned